package.json

```diff
@@ -26,6 +26,7 @@
     "eslint-plugin-react-refresh": "^0.4.9",
     "globals": "^15.9.0",
     "postcss": "^8.4.47",
+    "sass-embedded": "^1.79.4",
```

Comment:
why do we need sass/scss ?

this day and age, vanilla css supports nesting and variables , so there is no real reason imo to use scss